packages/command-palette/src/commands/use-single-site-commands.tsx

```diff
@@ -656,7 +645,7 @@ const useSingleSiteCommands = ( { navigate, currentRoute }: useCommandsParams ):
 				_x( 'create post', 'Keyword for the Add new post command', __i18n_text_domain__ ),
 				_x( 'write post', 'Keyword for the Add new post command', __i18n_text_domain__ ),
 			].join( ' ' ),
-			context: [ '/posts' ],
+			context: [ { path: '/wp-admin/edit.php', match: 'exact' } ],
```

Comment:
The `match` property is something I came up so the "Add new post" command does not appear at the top in other paths with the same root (e.g. `/wp-admin/edit.php?post_type=page`).